numpy_challenge.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+import numpy as np
+
+# Create three arrays
+ar1 = np.array([1, 2, 3])
+ar2 = np.arange(1, 4)
+ar3 = np.zeros(3) + ar1
+
+
+# Create matrix_multiplication
+def matrix_multiplication(a, b):
+    if a.shape[1] == b.shape[0]:
+        return np.dot(a, b)
+    else:
+        print('Parameters are not suitable for multiplication')
+        return None
+
+
+# Create multiplication_check
+def multiplication_check(matrix_list):
+    ret = True
+    n = len(matrix_list)
+    overlap = matrix_list[0].shape[1]
+    for i in range(1, n - 1):
+        if matrix_list[i].shape[0] == overlap:
+            overlap = matrix_list[i].shape[1]
+            continue
+        else:
+            ret = False
+            break
+    return ret
```

Comment:
Алгоритм верный, но можно упростить
```suggestion
    n = len(matrix_list)
    for i in range(n - 1):
        if matrix_list[i].shape[1] != matrix_list[i].shape[0]:
            return False
    return True
```